drift/info_parser.py

```diff
@@ -282,4 +282,5 @@ def _historical_sys_profile_mapping(historical_sys_profile):
         "id": historical_sys_profile["id"],
         "display_name": historical_sys_profile["display_name"],
         "updated": historical_sys_profile["updated"],
+        "inventory_id": historical_sys_profile["inventory_id"],
```

Comment:
I am leaning more towards `system_id` since we use the term `system` and `system_id` everywhere else that I could find in our APIs.

Reply:
i'll fix it up here. I'll change it later on in the HSP model obj